### DISCUSSION TITLE
Vercel Analytics for Remix apps?

### DISCUSSION THREAD

**Discussion by just214** [vercel/community#410] [Help] [answered]
I have a Remix app that is hosted on Vercel and I would really love to use analytics. Just curious if there is any timeline or priority on this feature?  Thanks!

**Comment by wiiiimm** [comment]
@gojutin thanks for your feedback. I have added it to our internal feature request list.

**Comment by leerob** [comment, marked as answer]
Hey! This is now possible using the [Vercel Analytics API](https://vercel.com/docs/concepts/analytics/api). You can view similar examples for [SvelteKit](https://github.com/vercel/vercel/tree/main/examples/sveltekit) and [Create React App](https://github.com/vercel/vercel/tree/main/examples/create-react-app) to see how it's implemented.

**Reply by just214** [reply]
A little more low-level than I was hoping for but I'll take it. Thanks!
